app/views/collections/show.html.erb

```diff
@@ -26,11 +26,11 @@
     <h3 class="collection_revision_header">Collection Revisions</h3>
     <% @collection_records.each_with_index do |record, index| %>
 
-      <div class="collection_revision">
+      <div id="collection_revision_<%= record.revision_id %>" class="collection_revision">
         <table cellpadding="5" border="1">
```

Comment:
Just from what I can see in this file, there are three instances of `<table cellpadding="5" border="1">`. I know you didn't add them but we discussed moving it into css. Maybe you can do it little by little, so add the class and use it for the tables when you happen to be making changes to those views?